src/App.tsx

```diff
@@ -1,26 +1,54 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
[Not critical]
Disabling eslint rules is not a big deal for a homework. But for a real work, I would not recommend to disable eslint rules if errors are produced on your code. These rules were most likely added for some reason, so it's better to fix your code to align it with rules. If this is not possible or you feel that a rule doesn't give enough profit, you can discuss changing its configuration or even removal with the team.

Reply:
thank you)
just for your information: this goes with the start code)

Reply:
Correct, I've seen that - and that's why this is not the reason for rejecting. This is more like advice for your future work.